instance/migrations/0053_instancemanagerpermission_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-07 06:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+from instance.permissions.models import InstanceManagerPermission
+
+def create_instance_manager_permission(apps, schema_editor):
+	"""Create the singleton InstanceManagerPermission"""
+	InstanceManagerPermission.objects.get_or_create(codename=InstanceManagerPermission._meta.verbose_name)
```

Comment:
Here is an alternative approach: https://gist.github.com/plar/5acaf6467e327dd3e478

In that code, the permissions are created when running the migrations, but not inside any permission.  However, they use the plain `Permission` model, and don't create a proxy model.  Not sure whether this could be adapted.

Treat these comments here with low priority.  I don't even know what problem I'm trying to solve here. :)